book/survival.qmd

```diff
@@ -195,7 +224,7 @@ In practice, when left- and right-censoring are both present, an additional vari
 This is often solved by removing the censoring indicator from the data and adding a second time column, then left-censored data is given by $(-\infty, A_i)$, right-censored data is denoted by $(A_i, \infty)$, and actual event times are $(T_i, T_i)$. -->
 
 
-#### Interval-censoring {.unnumbered .unlisted}
+### Interval-censoring {.unnumbered .unlisted}
```

Comment:
Should we say something in this section about interval censoring notation?

Reply:
I have kept the left-/interval censoring section without notation and introduce notation thoroughly in the section after

Reply:
Although we could just remove the "Censoring Notation" section header and just have one "Left- and Interval-censoring" section